unfurl/build.gradle.kts

```diff
@@ -5,8 +5,8 @@ plugins {
 apply(plugin = "com.vanniktech.maven.publish")
 
 dependencies {
-  api("com.squareup.okhttp3:okhttp:4.9.0")  // Updating to 5.x? See cli/build.gradle.kts.
-  api("org.jsoup:jsoup:1.16.1")
+  api("io.ktor:ktor-client-okhttp:2.3.4")
+  api("com.fleeksoft.ksoup:ksoup:0.0.2")
```

Comment:
Have you compared this with https://github.com/MohamedRejeb/Ksoup?

Reply:
Yes, this one does not have support for Element and Document, only Parsing hence would not be able to fulfill our purpose